lms/djangoapps/support/tests/test_views.py

```diff
@@ -78,6 +83,15 @@ def test_get_contact_us(self):
         response = self.client.get(url)
         assert response.status_code == 200
 
+    @override_settings(ZENDESK_URL=UNDEFINED_ZENDESK_URL)
+    def test_get_contact_us_redirect_if_undefined_zendesk_url(self):
+        """
+        Tests Support View contact us Page
```

Comment:
docstring can be improved(example: Tests the support contact us page redirects if Zendesk URL is not defined)